std_msgs/Header header
HandData[] recognizedHands
